### ISSUE TITLE
firewalld: firewalld page is in inconsistent page if enable firewall inside firewall tab

### ISSUE THREAD

**Issue by jscotka** [cockpit-project/cockpit#10501]
I've found that when machine has disabled ``firewalld`` and I enable it inside firewall tab,``on`` is marked grey, and adding service popup hangs. see screenshots. Version of cockpit: ``180-1``
It seems that there is missing some reload, or wait for firewall is really active.

![screenshot from 2018-11-05 08-14-32](https://user-images.githubusercontent.com/8735467/47983411-7f1b0a80-e0d3-11e8-8f74-886f413c8a53.png)
![screenshot from 2018-11-05 08-15-05](https://user-images.githubusercontent.com/8735467/47983430-8c37f980-e0d3-11e8-9d2d-77cc6846fcd0.png)
```
# systemctl status firewalld
● firewalld.service - firewalld - dynamic firewall daemon
   Loaded: loaded (/usr/lib/systemd/system/firewalld.service; enabled; vendor p>
   Active: active (running) since Mon 2018-11-05 02:14:38 EST; 16s ago
     Docs: man:firewalld(1)
 Main PID: 12862 (firewalld)
    Tasks: 2 (limit: 11534)
   Memory: 24.7M
   CGroup: /system.slice/firewalld.service
           └─12862 /usr/libexec/platform-python -s /usr/sbin/firewalld --nofork>

```
I had to ``cancel`` this operation
And when I reload page, then everything works as expected.
![screenshot from 2018-11-05 08-15-17](https://user-images.githubusercontent.com/8735467/47983432-8d692680-e0d3-11e8-9469-7d0208d1ccac.png)
![screenshot from 2018-11-05 08-15-22](https://user-images.githubusercontent.com/8735467/47983434-8e9a5380-e0d3-11e8-873a-a36885f2e39b.png)

**Comment by martinpitt** [comment]
The root cause of that behaviour is #9125 . This seems rather hard to fix, so we need to find a workaround.

**Comment by martinpitt** [comment]
Fixed in PR #10511

**Comment by martinpitt** [comment]
Filed downstream tracking bug https://bugzilla.redhat.com/show_bug.cgi?id=1646936
